src/lis3dhtr_constants.hpp

```diff
@@ -0,0 +1,32 @@
+#include <libhal/units.hpp>
+
+namespace hal::stm_imu {
+
+/// Device identification register
+static constexpr hal::byte who_am_i_register = 0x0F;
```

Comment:
```suggestion
constexpr hal::byte who_am_i_register = 0x0F;
```

I don't think static is required here. We should be able to remove these.